### PR TITLE
add unsubscribe functionality

### PR DESCRIPTION
**Pull request by hallarak**
Adding ability to unsubscribe from an action.

### CODE REVIEW

File: packages/satcheljs-stitch/lib/stitch.ts [hunk 1]
```diff
@@ -27,6 +27,16 @@ export function subscribe<T extends ActionHandler>(actionType: string, callback:
     handlers[actionType].push(callback);
 }
 
+export function unsubscribe<T extends ActionHandler>(actionType: string, callback: T) {
+    const actionHandlers = handlers[actionType] || [];
```
Comment (jair-cazarin) [inline packages/satcheljs-stitch/lib/stitch.ts]: why not return an unsubscribe function that closures the index from subscribe()? that's more idiomatic JS IMO.

**Comment by smikula** [comment]
Can you explain the rational behind adding this?  Not having an unsubscribe mechanism was an intentional choice.  (It would be akin to dynamically removing a reducer in Redux -- it doesn't really make sense.)

File: packages/satcheljs-stitch/lib/stitch.ts (hunk 1 above)
Comment (hallarak) [inline packages/satcheljs-stitch/lib/stitch.ts]: agreed.

**Comment by hallarak** [comment]
Closing this pull request as I will rather take a dependency on vNext.

**Pull request closed without merging**